server/api/event/tests.py

```diff
@@ -0,0 +1,130 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from .models import Event
+
+import datetime
+
+
+class EventTests(APITestCase):
+    def setUp(self):
```

Comment:
You'll also have to include the team FK in your tests

Reply:
oops, I forgot to do that, will fix thanks chantelle